config/base/crd/bases/olm.operatorframework.io_clusterextensions.yaml

```diff
@@ -574,6 +574,8 @@ spec:
                 required:
                 - bundle
                 type: object
+            required:
+            - conditions
```

Comment:
I would have expected no diff in our generated CRD by only adding the required tags to the already required fields

Reply:
Ditto! And yet ... 
It's even more apparently-empty in the generated docs!

Reply:
I left another comment that I think the conditions should be optional. We include `omitempty` in the JSON tag. If it should be required then this change is fine.